### PR TITLE
fix(unified-share-modal): increase pill selector height

### PR DESCRIPTION
**Pull request by nickgriffin1**
* Creates a global variable for pill selector height

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @nickgriffin1 has signed the CLA. Thanks for the pull request!

File: src/styles/variables.js [hunk 1]
```diff
@@ -75,4 +75,5 @@ export const menuZIndex = '170'; // menu-z-index
 export const notificationsWrapperZIndex = '180'; // notifications-wrapper-z-index
 export const overlayZIndex = '190'; // overlay-z-index
 export const tooltipZIndex = '190'; // tooltip-z-index
+export const defaultLineHeight = '20px'; // default-line-height
```
Comment (booc0mtaco) [inline src/styles/variables.js]: hmm, I would have expected the name to be `bdlLineHeight`. Did you add these two (the JS variable, and the corresponding JSON) manually?

Comment (nickgriffin1) [reply src/styles/variables.js]: This was automatically generated

Comment (nickgriffin1) [reply src/styles/variables.js]: On that point thought, I'm open to calling it either "default" or "global".

Comment (booc0mtaco) [reply src/styles/variables.js]: The name here and in the JSON should match a variable in _variables.scss, but I don't see one in here with the `default-line-height` name 🤔 (this happens in build-style-vars.js)

Can you re-run `build:variables` to double-check?

Comment (nickgriffin1) [reply src/styles/variables.js]: Sure I ran it again and got a diff on pretty much the whole file. I'll push that up so you can check it out.

Comment (nickgriffin1) [reply src/styles/variables.js]: Oddly enough, git locally gives me a much different diff than when I push to GHE. Might just be git being weird. Seems like it's the same as before.

**Pull request merged**